h-youjin/LV.0/2차원으로-만들기.js

```diff
@@ -0,0 +1,7 @@
+function solution(num_list, n) {
+  answer = [];
+  for (let i = 0; i < num_list.length; i + n) {
+    answer.push(num_list.splice(i, i + n));
+  }
+  return answer;
+}
```

Comment:
```suggestion
function solution(num_list, n) {
  const answer = [];
  
  for (let i = 0; i < num_list.length; i + n) {
    answer.push(num_list.splice(i, i + n));
  }
  
  return answer;
}
```
- 변수 선언 키워드가 빠졌네요!!
- 만약 인자로 넘겨받은 num_list 라는 배열을 이용 시 splice 하게되면 함수 밖 다른 곳에서 num_list를 이용하고 있었다면 오류가 발생했을지도 모릅니다..!
- 이때는 num_list를 깊은복사하여 사용하는 방법이 있습니다.